docs/Guidelines/GettingStarted/gettingstarted.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { Meta } from '@storybook/addon-docs/blocks'
+
+<Meta
+  parameters={{
+    previewTabs: {
+      'storybook/docs/panel': {
+        hidden: true,
+      },
+    },
+  }}
+  title="Guidelines/Getting started"
+/>
+
+[![Actions Status](https://github.com/TicketSwap/solar/workflows/Continuous%20Integration/badge.svg)](https://github.com/TicketSwap/solar/actions)
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+
+# Solar
+
+This documents the work on the Solar product, our TicketSwap Design System. Solar is set of UI components that form the foundation of TicketSwap’s design language. By utilising these building blocks we’re able to standardise the user experience across our platform.
+
+## Installation
+
+**The library assumes the consuming application uses React.js v16+.**
```

Comment:
If we mention this, we should also mention the other peer dependencies imo or don't mention this at all.

Reply:
What would those be @glenngijsberts ?

Reply:
You can find those in the `package.json` under the `peerDependencies` :)